sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/sweep/early_termination_policy.py

```diff
@@ -192,15 +173,6 @@ def __init__(
         evaluation_interval: Optional[int] = 0,
         truncation_percentage: Optional[int] = 0,
     ) -> None:
-        """Defines an early termination policy that cancels a given percentage of runs at each evaluation interval.
```

Comment:
It feels odd to remove the init docstrings for all the different initializers here, especially since some of them have different parameters than that's listed in the class-wide docstring. Is there some sort of documentation edge case that applies to classes with multiple initializers with differing parameters?

Reply:
What are cases where there are different parameters than the class-wide docstring?

The init docstring and the class docstring will should up as a duplicate if you have docstrings for both. One needs to be definitive. In some of the cases where they are different, it was a typo or the __init__ method not including certain params that are ultimately consumed by a parent class even though those params are taken in. If you point to some specific cases, then I case see if there's an edge case I'm missing.